main.go

```diff
@@ -72,20 +100,21 @@ type Workspace struct {
 
 // Client represents a client.
 type Client struct {
-	Wid   int    `json:"wid"`
-	ID    int    `json:"id"`
-	Name  string `json:"name"`
-	Notes string `json:"notes"`
+	Wid      int    `json:"wid"`
+	ID       int    `json:"id"`
+	Name     string `json:"name"`
+	Archived bool   `json:"archived"`
+	Notes    string `json:"notes"`
 }
 
 // Project represents a project.
 type Project struct {
 	Wid             int        `json:"wid"`
 	ID              int        `json:"id"`
-	Cid             int        `json:"cid"`
+	Cid             *int       `json:"cid,omitempty"`
 	Name            string     `json:"name"`
 	Active          bool       `json:"active"`
-	Billable        bool       `json:"billable"`
+	Billable        *bool      `json:"billable,omitempty"`
```

Comment:
For users without Pro account, it was impossible to do the update, because API validation enforces checks on this field.

Reply:
At least the API docs type that one as `integer | null`. They suggest that `cid` will always be an integer, but they lie. 🙂